src/rules/no-large-snapshots.ts

```diff
@@ -23,7 +25,10 @@ type RuleContext = TSESLint.RuleContext<MessageId, [RuleOptions]>;
 const reportOnViolation = (
   context: RuleContext,
   node: TSESTree.CallExpression | TSESTree.ExpressionStatement,
-  { maxSize: lineLimit = 50, whitelistedSnapshots = {} }: RuleOptions,
+  {
```

Comment:
I think it'll be best to break this into two arguments: that'll work better with my above comment about using `maxSize` & `inlineMaxSize`, since you'll be able to just do `options.maxSize` & `options.inlineMaxSize ?? options.maxSize` depending on the branch :)

Reply:
@leonardovillela this is definitely a refactor we should do, as it cuts out the need for `buildOptionsByMaxSizeAttributeName` entirely :)